media-sound/zynaddsubfx/zynaddsubfx-3.0.6-r1.ebuild

```diff
@@ -7,12 +7,21 @@ inherit cmake flag-o-matic
 
 DESCRIPTION="Software synthesizer capable of making a countless number of instruments"
 HOMEPAGE="https://zynaddsubfx.sourceforge.net/"
-SRC_URI="mirror://sourceforge/zynaddsubfx/${P}.tar.bz2"
+
+# The mruby-zest-build archive was made by:
+#   $ git clone --recurse-submodules -b ${PV} https://github.com/mruby-zest/mruby-zest-build
+#   $ cd mruby-zest-build
+#   $ git ls-files --recurse-submodules | tar caf ../mruby-zest-build-${PV}.tar.gz --xform s:^:zynaddsubfx-${PV}/mruby-zest-build/: --verbatim-files-from -T-
+#   $ scp ../mruby-zest-build-${PV}.tar.gz dev.gentoo.org:public_html/distfiles/
+SRC_URI="
+	mirror://sourceforge/zynaddsubfx/${P}.tar.bz2
+	https://dev.gentoo.org/~zx2c4/distfiles/mruby-zest-build-${PV}.tar.gz
```

Comment:
Repackaging this isn't necessary. The full UI sources including submodules are in `mirror://sourceforge/zynaddsubfx/zyn-fusion-ui-src-${PV}.tar.bz2`

Reply:
Thanks!